src/postgres/tree/PostgresDatabaseTreeItem.ts

```diff
@@ -23,6 +23,7 @@ export class PostgresDatabaseTreeItem extends AzureParentTreeItem<ISubscriptionC
     public static contextValue: string = "postgresDatabase";
     public readonly contextValue: string = PostgresDatabaseTreeItem.contextValue;
     public readonly childTypeLabel: string = "Resource Type";
+    public connectionString: string;
```

Comment:
The type `string` here isn't accurate, because this will be undefined until people call `getClientConfig`. It's also not clear that `getClientConfig` would populate the connection string. One possible fix is to remove `connectionString` from this class and have `copyConnectionString.ts` use the clientConfig to construct the connection string in it's own file